integration-tests/tests/run-notebook.js

```diff
@@ -1,3 +1,6 @@
+// The notebook run-regional-notebook.js is intended to be the same as this,
+// but with additional regional requirements. Generally any changes to this
+// file should also be made in run-regional-notebook.js
```

Comment:
Is it possible to factor out the common code?

Reply:
Yes, please do not duplicate the code. And is it necessary to do the whole test with the different location, or do you just want to verify a subset of the functionality?